R/groundwater_recharge.R

```diff
@@ -1,58 +1,144 @@
 #' Calculate groundwater recharge of a soil
 #' 
-#' This function calculates an index score for groundwater storage based on WHC, sealing risk, drainage and subsoil compaction
+#' This function calculates an index score for groundwater storage based on precipitation surplus, infiltration at saturation, sealing risk, drainage and subsoil compaction
 #' 
 #' 
-#' @param D_WRI_WHC (numeric) The value for Water Retention index (WRI) for WHC as calculated by \code{\link{calc_waterretention}}
+#' @param B_LU_BRP (numeric) The crop code from the BRP
 #' @param D_PSP (numeric) The precipitation surplus per crop  calculated by \code{\link{calc_psp}}
+#' @param D_WRI_K (numeric) The value for top soil permeability (cm/d) as calculated by \code{\link{calc_permeability}}
 #' @param I_P_SE (numeric) The indicator value for soil sealing
 #' @param I_P_CO (numeric) The indicator value for occurrence of subsoil compaction
 #' @param B_DRAIN (boolean) Are drains installed to drain the field (options: yes or no)
+#' @param B_GWL_CLASS (character) The groundwater table class
 #' 
 #' @examples 
-#' ind_gw_recharge(D_WRI_WHC = 0.6, D_PSP = 200, I_P_SE = 0.6, I_P_CO = 0.9)
-#' ind_gw_recharge(D_WRI_WHC = 0.8, D_PSP = 400, I_P_SE = 0.4, I_P_CO = 0.2)
+#' ind_gw_recharge(B_LU_BRP = 265,D_PSP = 200, D_WRI_K = 10, I_P_SE = 0.6, I_P_CO = 0.9, 
+#' B_DRAIN = FALSE, B_GWL_CLASS = 'GtV')
+#' ind_gw_recharge(B_LU_BRP = 233, D_PSP = 400, D_WRI_K = 10, I_P_SE = 0.4, I_P_CO = 0.2, 
+#' B_DRAIN = TRUE, B_GWL_CLASS = 'GtII')
 #'
 #' @return 
 #' The evaluated score for the soil function to improve groundwater recharge. A numeric value between 0 and 1.
 #'          
 #' @export
-ind_gw_recharge <- function(D_WRI_WHC, D_PSP, I_P_SE, I_P_CO, B_DRAIN = FALSE){
+ind_gw_recharge <- function(B_LU_BRP, D_PSP, D_WRI_K, I_P_SE, I_P_CO, B_DRAIN, B_GWL_CLASS){
```

Comment:
This looks like a breaking change. It would be good to inform users about this in the changelog and use major version bump

Reply:
agreed. note that this are indeed functions that are not used in the OBIC wrapper. But since its a change given the main branch, this should be mentioned.